module/TaskManagement/public/js/task-detail.js

```diff
@@ -0,0 +1,143 @@
+var TaskDetail = function(taskUtils){
```

Comment:
è un file nuovo che ho aggiunto per fattorizzare in un unico punto la visualizzazione della popup con i dettagli di un task